src/lib/rest/OrionError.cpp

```diff
@@ -60,12 +60,12 @@ OrionError::OrionError(HttpStatusCode _code, const std::string& _details)
 
 /* ****************************************************************************
 *
-* OrionError::OrionError - 
+* OrionError::OrionError -
 */
-OrionError::OrionError(HttpStatusCode _code, std::string& _details)
+OrionError::OrionError(const std::string _reasonPhrase, const std::string& _details)
 {
-  code          = _code;
-  reasonPhrase  = httpStatusCodeString(code);
+  code          = SccNone;
+  reasonPhrase  = _reasonPhrase;
   details       = _details;
 }
```

Comment:
So, you remove the code, setting it no SccNone always?

Not sure this is a good idea ...